encrypted-media-respec.html

```diff
@@ -887,42 +926,84 @@
+              <var>accumulated configuration</var>. Previous consent for <var>accumulated configuration</var> with <a def-id="option-distinctiveIdentifier"></a>
+              set to <a def-id="requirement-not-allowed"></a> does not imply consent for the same configuration with <a def-id="option-distinctiveIdentifier"></a>
+              set to <a def-id="requirement-optional"></a> or <a def-id="requirement-required"></a>.
             </p>
             <ol>
               <li>
                 <p>
-                  If there is persisted denial for <var>origin</var> indicating that <a def-id="distinctive-identifier">Distinctive Identifier(s)</a> are not allowed in conjunction with <var>accumulated configuration</var>, return <code>Denied</code>.
+                  If there is persisted denial for <var>origin</var> indicating that <var>accumulated configuration</var> is not allowed:
                 </p>
+                <ol>
+                  <li>
+                    <p>Add all the configurations for which consent has been denied to <var>restrictions</var>.</p>
+                  </li>
+                  <li>
+                    <p>Return <code>ConsentDenied</code> and <var>restrictions</var>.</p>
+                  </li>
+                </ol>
               </li>
               <li>
                 <p>
-                  If there is persisted consent for <var>origin</var> indicating that <a def-id="distinctive-identifier">Distinctive Identifier(s)</a> are allowed in conjunction with <var>accumulated configuration</var>, return <code>Allowed</code>.
-                </p>
-                <p class="note">
-                  A previous user consent for a prompt that did not include use of a <a def-id="distinctive-identifier"></a> (with similar properties) would not be considered as covering an <var>accumulated configuration</var> that requires use of such an identifier.
+                  If there is persisted consent for <var>origin</var> indicating <var>accumulated configuration</var> is allowed, return <code>Allowed</code>.
                 </p>
               </li>
               <li>
                 <p>
-                  If any of the recommendations of <a href="#allow-identifiers-cleared">Allow Identifiers to be Cleared</a> are not supported by the User Agent, return <code>ConsentRequired</code>.
+                  If any of the following are true:
                 </p>
+                <ul>
+                  <li>
+                    <p>
+                      the <a def-id="option-distinctiveIdentifier"></a> member of <var>accumulated configuration</var> is not
+                      <a def-id="requirement-not-allowed"></a> and any of the recommendations of
+                      <a href="#allow-identifiers-cleared">Allow Identifiers to be Cleared</a> are not supported by the
+                      User Agent.
```

Comment:
I think this should be "implementation" since it also includes the CDM. (It appears we use these words somewhat interchangeably and inconsistently, but here we should cover both the UA and CDM.)